configure.ac

```diff
@@ -929,6 +929,13 @@ fi
 
 CPPFLAGS="-I./src $CPPFLAGS"
 
+# Remove unnecessary whitespaces
+LDFLAGS="`$ECHO \"$LDFLAGS\" | $SED 's/ \+/ /g; s/ $//'`"
+LIBS="`$ECHO \"$LIBS\" | $SED 's/ \+/ /g; s/ $//'`"
+CFLAGS="`$ECHO \"$CFLAGS\" | $SED 's/ \+/ /g; s/ $//'`"
+CPPFLAGS="`$ECHO \"$CPPFLAGS\" | $SED 's/ \+/ /g; s/ $//'`"
+TESTCFLAGS="`$ECHO \"$TESTCFLAGS\" | $SED 's/ \+/ /g; s/ $//'`"
+
```

Comment:
Doesn't this break quoting insider arguments, eg when a an include path contains spaces?

why is this even necessary?

Reply:
Oh, it's not. Personally I just do not like to have unnecessary whitespaces in the flags.

Reply:
I will fix the quotation.